lib/Config.php

```diff
@@ -260,35 +260,51 @@ public function getTurnSettings(): array {
 
 		if (empty($servers)) {
 			return [
-				'schemes' => '',
-				'server' => '',
-				'username' => '',
-				'password' => '',
-				'protocols' => '',
+				'turnservers' => [
+					'schemes' => '',
+					'server' => '',
+					'username' => '',
+					'password' => '',
+					'protocols' => '',
+				]
 			];
 		}
 
-		// For now we use a random server from the list
-		try {
-			$server = $servers[random_int(0, count($servers) - 1)];
-		} catch (\Exception $e) {
-			$server = $servers[0];
-		}
+		// Randomize server list
+		shuffle($servers);
```

Comment:
```suggestion
		// Randomize server list
		shuffle($servers);
```
lets not do that, so we have more reliable results and not a F5 changing something.

Reply:
What would you suggest?

Right now with _one_ random server chosen by spreed and with multiple servers specified in the backend you suffer from the same "F5 changing something".